.travis.yml

```diff
@@ -11,7 +11,7 @@ env:
     - K8S_VERSION="v1.19.0"
     - MINIKUBE_VERSION="v0.28.2"
     - IMAGE_NAME=storageos/cluster-operator
-    - IMAGE_TAG=test
```

Comment:
Previously for `operator-image` target we used `IMAGE_TAG` but for `install-manifest` we used `TRAVIS_TAG` and then we re-tagged image. With this change we build only `TRAVIS_TAG` so w save an extra tagging.